WebRoot/public/login_history.jsp

```diff
@@ -0,0 +1,105 @@
+<style>
```

Comment:
Please add directives of buffer, session, autoFlush, pageEncoding, contentType, charset etc.

Reply:
please move style block to a css file

Reply:
Please add license block